Tests/Testably.Abstractions.Tests/FileSystem/FileSystemWatcher/NotifyFiltersTests.cs

```diff
@@ -654,11 +654,13 @@ public async Task NotifyFilter_MoveDirectory_ShouldTriggerChangedEventOnNotifyFi
 	}
 
 	[Theory]
-	[InlineAutoData(NotifyFilters.DirectoryName)]
-	public async Task NotifyFilter_MoveDirectoryOutOfTheWatchedDirectory_ShouldTriggerChangedEventOnNotifyFilters(
-		NotifyFilters notifyFilter, string sourceName, string destinationName)
+	[InlineAutoData(NotifyFilters.DirectoryName, true)]
+	[InlineAutoData(NotifyFilters.DirectoryName, false)]
+	public async Task NotifyFilter_MoveDirectoryOutOfTheWatchedDirectory_ShouldTriggerDeletedEventOnNotifyFilters_OnWindows(
+		NotifyFilters notifyFilter, bool includeSubdirectories, string sourceName, string destinationName)
 	{
 		SkipIfLongRunningTestsShouldBeSkipped();
+		Skip.IfNot(Test.RunsOnWindows);
```

Comment:
Correcting the event that should be triggered, marking as Windows-only, and testing both true and false for `IncludeSubDirectories`.